src/palace.pyx

```diff
@@ -2351,12 +2352,18 @@ cdef class ChorusEffect(BaseEffect):
         self.slot.apply_effect(self.impl)
 
     @property
-    def waveform(self) -> int:
-        return self.properties.waveform
+    def waveform(self) -> str:
+        if self.properties.waveform == 1:
+            return 'triangle'
+        else:
+            return 'sin'
 
     @waveform.setter
-    def waveform(self, value: int) -> None:
-        self.properties.waveform = value
+    def waveform(self, value: str) -> None:
+        if value == 'triangle':
+            self.properties.waveform = 1
+        else:
+            self.properties.waveform = 0;
```

Comment:
There should be no semicolon here.

Reply:
LOL I was staring and having no idea where the semi colon at. Here is how it look on my editor
![Screenshot from 2020-04-28 19-46-54](https://user-images.githubusercontent.com/48543141/80488805-1f7c8300-8989-11ea-9d41-ee0e8e6a8bf7.png)

Reply:
Lol that's because the color highlighting for semicolon is too so similar to the background.